stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1653,7 +1653,8 @@ function(add_swift_target_library name)
     list(APPEND SWIFTLIB_SWIFT_COMPILE_FLAGS "-warn-implicit-overrides")
   endif()
 
-  if(NOT SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER AND NOT BUILD_STANDALONE)
+  if(NOT SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER AND NOT BUILD_STANDALONE AND
+     NOT SWIFT_PREBUILT_CLANG)
```

Comment:
@gottesmm, I'm guessing you passed in `--build-runtime-with-host-compiler=1` to the standalone stdlib preset only to keep this from erroring out? This and the next change make that flag no longer necessary.